cookbooks/aws-parallelcluster-platform/resources/nvidia_dcgm/nvidia_dcgm_ubuntu20+.rb

```diff
@@ -21,3 +21,7 @@
 def _nvidia_dcgm_enabled
   _nvidia_enabled
 end
+
+def package_version
+  "1:#{node['cluster']['nvidia']['dcgm_version']}"
```

Comment:
is the string "1: [...]" correct?
Asking because I don't see it matching the package in the repo at https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/

Reply:
To verify this changes is useful to run `kitchen.ec2.sh platform-install test nvidia-dcgm-ubuntu20` locally. Did you run it?

Reply:
I didn't run kitchen yesterday. I run it today and it passed.

The single digit "1" is epoch version. Without the "1", package install fails because version does not exist. See details here: https://askubuntu.com/questions/441879/why-do-some-packages-have-extra-numbers-before-a-colon-on-the-front-of-their-ver